tests/tests/tier0/bluechi-reload-unit-service/service/new-service.service

```diff
@@ -0,0 +1,12 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
Just for consistencies sake:
Could your rename the directory `service -> systemd`? Its called like this in all other tests.